### PR TITLE
Links to relevant documentation

### PR DESCRIPTION
**Pull request by rtmalone**
Links to React, Webpack, Babel, and Jest documentation

Relevant to #26

### CODE REVIEW

File: src/components/Body.js [hunk 1]
```diff
@@ -6,6 +6,13 @@ const Body = props => {
     <div className="bg-light">
```
Comment (kbrock84) [inline src/components/Body.js]: The links are all showing on one line with no spaces between:
image

Try adding a link style like 
``` javascript
const linkStyle = {
  display: "block"
}

```

File: src/components/Body.js [hunk 2]
```diff
@@ -6,6 +6,13 @@ const Body = props => {
...
       <div className="body">
         <p className="tx-dark">{props.text}</p>
+        <p className="tx-dark">These links may be helpful:</p>
```
Comment (kbrock84) [inline src/components/Body.js]: Lets make this an H2 tag or something similar:
```suggestion
        <h2 className="tx-dark">These links may be helpful:</h2>
```

File: src/components/Body.js [hunk 3]
```diff
@@ -6,6 +6,13 @@ const Body = props => {
...
+        <a href="https://reactjs.org/docs/getting-started.html">
+          React Documention
+        </a>
+        <a href="https://webpack.js.org/concepts/">Webpack Documention</a>
```
Comment (kbrock84) [inline src/components/Body.js]: And then in the link:
```suggestion
        <a style={linkStyle} href="https://webpack.js.org/concepts/">Webpack Documention</a>
```

**Comment by rtmalone** [comment]
Ok, I added some better styling.

One suggestion...I'd remove zeroing out the padding and margin on the whole document with `*`. In my opinion breaking that style for all elements in the document is bad practice. There are better ways to normalize styles across browsers. Just a suggestion since it's not my repo.

**Comment by kbrock84** [comment]
> Ok, I added some better styling.
> 
> One suggestion...I'd remove zeroing out the padding and margin on the whole document with `*`. In my opinion breaking that style for all elements in the document is bad practice. There are better ways to normalize styles across browsers. Just a suggestion since it's not my repo.

I created an issue #33 for this. Thanks for your suggestion.

**Pull request merged**